vnext/Scripts/Tfs/Layout-Headers.ps1

```diff
@@ -2,8 +2,8 @@
 # Licensed under the MIT License.
 
 param(
-	[string] $TargetRoot = "$SourceRoot\vnext\target",
 	[string] $SourceRoot = ($PSScriptRoot | Split-Path | Split-Path | Split-Path),
+	[string] $TargetRoot = "$SourceRoot\vnext\target",
```

Comment:
Nit: revert this file?

Reply:
This fix is actually used for generating the OpenSSL NuGet.